.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-  "editor.formatOnSave": true
-}
+  "editor.formatOnSave": false,
```

Comment:
I had to disable this because it was breaking the build on a space it was adding between the {} on https://github.com/sonatype-nexus-community/auditjs/pull/186/files#diff-e2206ba0c17727f95c268ab7b1f899e3R22

Reply:
Shrug. I'm still using IDEA. ;)

Reply:
^^^ noooooob.

Just kidding :)

I'll try and pull this down and take a gander at why it was breaking the build.